### PR TITLE
[MKT-177]: feat/create Cyber Awareness banner

### PR DESCRIPTION
**Pull request by xabg2**
A new banner has been added to promote the cyber awareness campaign. The banner redirects the user to our pricing page (website).

### CODE REVIEW

**Comment by xabg2** [comment]
⚠️ Merge on October 2nd.

File: src/app/drive/components/DriveExplorer/DriveExplorer.tsx [hunk 1]
```diff
@@ -1047,6 +1049,7 @@ const DriveExplorer = (props: DriveExplorerProps): JSX.Element => {
             </>
           )}
         </div>
+        Ba
```
Comment (CandelR) [inline src/app/drive/components/DriveExplorer/DriveExplorer.tsx]: this text has slipped in

File: src/app/core/services/local-storage.service.ts [hunk 2]
```diff
@@ -6,6 +6,7 @@ export const STORAGE_KEYS = {
   SIGN_UP_TUTORIAL_COMPLETED: 'signUpTutorialCompleted',
   SHOW_LIFETIME_BANNER: 'showLifetimeOffer',
   SHOW_CROWDCUBE_BANNER: 'showCrowdcubeBanner',
```
Comment (CandelR) [inline src/app/core/services/local-storage.service.ts:8]: if this `SHOW_CROWDCUBE_BANNER` is not gonna be used anymore remove it

Comment (xabg2) [reply src/app/core/services/local-storage.service.ts:8]: I don't know if we will use the banner or not

File: src/app/drive/components/DriveExplorer/DriveExplorer.tsx (hunk 1 above)
Comment (xabg2) [reply src/app/drive/components/DriveExplorer/DriveExplorer.tsx]: Ops

**Comment by xabg2** [comment]
We can deploy the banner @CandelR

File: src/app/banners/CyberAwarenessBanner.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+                    color: '#001D6C',
+                  }}
+                >
+                  80% OFF
+                </p>
+              </div>
+              <button
+                className="focus:outline-none flex flex-row items-center justify-center space-x-4 rounded-lg bg-white py-3 px-5 text-base font-medium text-gray-80 transition duration-100 focus-visible:bg-gray-10 active:bg-gray-10 sm:text-lg"
+                onClick={() => {
+                  window.open(redirectURL, '_blank');
```
Comment (CandelR) [inline src/app/banners/CyberAwarenessBanner.tsx]: add noopener attribute

**Review by CandelR — APPROVED** [review]

**Pull request merged**